matrix-sdk-android/src/main/java/org/matrix/android/sdk/internal/auth/db/AuthRealmMigration.kt

```diff
@@ -17,28 +17,26 @@
 package org.matrix.android.sdk.internal.auth.db
 
 import io.realm.DynamicRealm
-import io.realm.RealmMigration
 import org.matrix.android.sdk.internal.auth.db.migration.MigrateAuthTo001
 import org.matrix.android.sdk.internal.auth.db.migration.MigrateAuthTo002
 import org.matrix.android.sdk.internal.auth.db.migration.MigrateAuthTo003
 import org.matrix.android.sdk.internal.auth.db.migration.MigrateAuthTo004
 import org.matrix.android.sdk.internal.auth.db.migration.MigrateAuthTo005
-import timber.log.Timber
+import org.matrix.android.sdk.internal.util.database.MatrixRealmMigration
 import javax.inject.Inject
 
-internal class AuthRealmMigration @Inject constructor() : RealmMigration {
+internal class AuthRealmMigration @Inject constructor() : MatrixRealmMigration(
+        dbName = "Auth",
+        schemaVersion = 5L,
+) {
     /**
      * Forces all AuthRealmMigration instances to be equal.
      * Avoids Realm throwing when multiple instances of the migration are set.
      */
     override fun equals(other: Any?) = other is AuthRealmMigration
     override fun hashCode() = 4000
 
-    val schemaVersion = 5L
-
-    override fun migrate(realm: DynamicRealm, oldVersion: Long, newVersion: Long) {
-        Timber.d("Migrating Auth Realm from $oldVersion to $newVersion")
-
+    override fun doMigrate(realm: DynamicRealm, oldVersion: Long) {
```

Comment:
`newVersion` is never used, also I think it will be always equal to `schemaVersion` if it is necessary.